File: 백준/Bronze/13129. Disposable Cup/Disposable Cup.java
import java.io.*;

class Main {
    static int nextInt() throws IOException {
        int c, n = System.in.read() & 15;
        while ((c = System.in.read()) > 32) {
            n = (n << 3) + (n << 1) + (c & 15);
        }
        return n;
    }

    public static void main(String[] args) throws IOException {
        int A = nextInt(), B = nextInt(), N = nextInt();
        StringBuilder sb = new StringBuilder();

        int min = A * N + B;
        sb.append(min).append(' ');
        for (int i = 1; i < N; i++) {
            min += B;
            sb.append(min).append(' ');
        }
        System.out.print(sb);
    }
}

File: 백준/Bronze/13129. Disposable Cup/Disposable Cup.rs
use std::io::{self, Read};

fn main() {
    let mut buffer = String::new();
    io::stdin().read_to_string(&mut buffer).unwrap();
    let mut it = buffer.split_whitespace();

    let a: i32 = it.next().unwrap().parse().unwrap();
    let b: i32 = it.next().unwrap().parse().unwrap();
    let n: i32 = it.next().unwrap().parse().unwrap();

    let mut min = a * n + b;
    let mut out = String::new();
    out.push_str(&min.to_string());
    out.push(' ');

    for _ in 1..n {
        min = min + b;
        out.push_str(&min.to_string());
        out.push(' ');
    }
    print!("{out}");
}